bin/clean_all

```diff
@@ -20,7 +20,7 @@ top_dir=$(cd $(dirname $(dirname $0)) && pwd)
 set -x
 set -e
 
-for pkg in signing cli sdk sdk/python validator docs core extensions/arcade core_transactions rest_api \
+for pkg in signing cli sdk sdk/python validator docs extensions/arcade core_transactions rest_api \
```

Comment:
It would be nice to have clean_all clean up 0.7 stuff as well when switching between 0.7 and 0.8.